### ISSUE TITLE
Support Kotlin Tests

### ISSUE THREAD

**Issue by ScottPierce** [graalvm/native-build-tools#57]
The native tests only seem to run when explicitly using `org.junit.jupiter.api.Test` annotations. Kotlin provides a testing library that has it's own test annotations: `kotlin.test.Test`, which is just a typealias for junit test annotation, which means when the code is compiled, the compiler swaps in `org.junit.jupiter.api.Test` in placce of `kotlin.test.Test`:
```Kotlin
public actual typealias Test = org.junit.jupiter.api.Test
```

However, if I use `kotlin.test.Test`, when I run the `nativeTest` command, all I see is:
```
[junit-platform-native] Running in 'test listener' mode.
```
and the tests never run.

**Comment by lazar-mitrovic** [comment]
Would you mind creating a reproducer for this one? 
I'm not too experienced with Kotlin but AFAIK if annotations are truly aliased we should be able to extract them from the bytecode.

My guess would be that this issue is related to the Gradle plugin itself hooking onto Java source sets (that are missing in this case).

**Comment by lazar-mitrovic** [comment]
#71 features explicit Kotlin tests, so any such issue (if present) should be fixed by `0.9.2` release.
